core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -728,6 +728,8 @@ private void handleLocaleCheck() {
     tabCallback = null;
     hideBackToTopTimer.cancel();
     hideBackToTopTimer = null;
+    webViewList.clear();
+    actionBar = null;
```

Comment:
do we need to set the actionbar to null?

Reply:
Yes, any view reference should be nulled before view destruction. I verified again, with the steps described above and got the following leak. If `actionbar` is set to null in `onDestroyView`, the leak is not reproducible. 
![118593919_1190825684606355_9144095146963418603_n](https://user-images.githubusercontent.com/22193232/91466965-2d3b0a00-e890-11ea-9bf3-352c18647b75.jpg)